### DISCUSSION TITLE
Merging remote repo with canvas changes - tips?

### DISCUSSION THREAD

**Discussion by Smekl** [Vinzent03/obsidian-git#738] [Problems]
very often I am working with an obsidian repo that has canvas (I am a heavy canvas user) and seems like git is not able to merge simple canvas changes. even if it just adding a new node, it will still fail to merge and I would have to do it manually.
quite frankly it gets annoying pulling / pushing stuff like this.

do guys have suggestions what can I do to cope with this?
thanks

**Comment by Vinzent03** [comment]
Obsidian Canvas are just json files, but I don't think there are any tricks to improve that workflow.
